### PR TITLE
Removing the use of functions

### PR DESCRIPTION
**Pull request by rschwabco**

### CODE REVIEW

**Comment by authereal** [comment]
@ogazitt do we want the policy file naming convention to be `<package name>.rego`?

**Comment by authereal** [comment]
can we remove the `"peoplefinder.IMPERSONATE.api.users.__id"` sections in `roles/data.json`? looks like the policy was already removed: https://github.com/aserto-demo/policy-peoplefinder/commit/f7fbeb51e4085351168322b3108249c617343f3e

File: src/roles/data.json [hunk 1]
```diff
@@ -1,124 +1,109 @@
 {
-    "roles": {
-        "viewer": {
```
Comment (ogazitt) [inline src/roles/data.json:3]: I'm not sure why this file changed?

File: src/policies/__id/peoplefinder.DELETE.api.users.__id.rego [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package peoplefinder.DELETE.api.users.__id
+
+import input.policy.path
+import input.user.applications.peoplefinder
```
Comment (ogazitt) [inline src/policies/__id/peoplefinder.DELETE.api.users.__id.rego]: we're no longer using input.user.applications.peoplefinder - we should be using input.user.attributes

File: src/policies/__id/peoplefinder.DELETE.api.users.__id.rego [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package peoplefinder.DELETE.api.users.__id
```
Comment (ogazitt) [inline src/policies/__id/peoplefinder.DELETE.api.users.__id.rego]: renaming all the files is a convention change - I'm not against it per se, but it should be done as a joint decision. adding @gertd for his opinion.

**Review by ogazitt — CHANGES_REQUESTED** [review]

File: src/roles/data.json (hunk 1 above)
Comment (rschwabco) [reply src/roles/data.json:3]: I removed the "Impersonate" entries.

Comment (ogazitt) [reply src/roles/data.json:3]: 👍

**Review by ogazitt — APPROVED** [review]

**Pull request merged**